### DISCUSSION TITLE
mt7981 Support / GL.Inet Beryl AX MT-3000

### DISCUSSION THREAD

**Discussion by calk1calk** [SmoothWAN/SmoothWAN#178] [Ideas and feature requests]
I'd love to permanently use SmoothWAN one on of my newer GL.Inet Routers like the Beryl AX (mt7981 soc)

Is there any way to get a build?

Thanks a lot!

**Comment by TalalMash** [comment]
https://github.com/TalalMash/Unofficial-Speedify-Installer-For-OpenWrt is under testing but it should work.
